app/views/users/_user_info.html.erb

```diff
@@ -0,0 +1,7 @@
+<article class="users-article">
+<div class="photo"><%= user.photo %></div>
```

Comment:
- [ ] As suggested by the previous code reviewer to show the image of the user in all the views, Kindly use img tag to show the image of the user in all the views including here 👍